primitives/weights/src/weight_v2.rs

```diff
@@ -33,12 +33,15 @@ use super::*;
 	Clone,
 	RuntimeDebug,
 	Default,
-	CompactAs,
 )]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub struct Weight {
+	#[codec(compact)]
```

Comment:
This will need a migration?

Reply:
yes. but the whole PR will need migration, since weight is now 2-dimensional, so good to have both of these changes in at once.